keyboards/handwired/nanggong/info.json

```diff
@@ -0,0 +1,126 @@
+{
+    "manufacturer": "Sanma",
+    "keyboard_name": "Nanggong",
+    "maintainer": "Sanma",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "build": {
+        "lto": true
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "rgblight": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B3", "B2", "B6", "B5", "D7", "E6", "B4", "C6", "D4", "D0", "D1", "D2", "D3"],
+        "rows": ["D5", "F5", "F6", "F7", "B1"]
+    },
+    "usb": {
+        "vid": "0x736D",
+        "pid": "0x0002",
+        "device_version": "1.0.1"
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "rgblight": {
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "led_count": 8,
+        "sleep": true,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": false,
+            "knight": true,
+            "christmas": false,
+            "static_gradient": true,
+            "rgb_test": true,
```

Comment:
```suggestion
```

Reply:
@fauxpark Thank sir for checking my PR. I have made the changes according to your review. Thank you.